src/OpenTelemetry.Instrumentation.Http/Implementation/HttpHandlerDiagnosticListener.cs

```diff
@@ -184,6 +184,8 @@ public override void OnException(Activity activity, object payload)
                     HttpInstrumentationEventSource.Log.EnrichmentException(ex);
                 }
 
+                activity.RecordException(exc);
```

Comment:
lets discuss and do this in separate PR.

Reply:
ok. let me revert then